spec/formatting.md

```diff
@@ -1,24 +1,113 @@
-# WIP DRAFT MessageFormat 2.0 Formatting Behaviour
+# DRAFT MessageFormat 2.0 Formatting Behaviour
 
 ## Introduction
 
 This document defines the behaviour of a MessageFormat 2.0 implementation
 when formatting a message for display in a user interface, or for some later processing.
 
-The document is part of the MessageFormat 2.0 specification,
-the successor to ICU MessageFormat, henceforth called ICU MessageFormat 1.0.
+Message formatting is defined in terms of the following stages:
```

Comment:
I'm not sure if "stages" is the right way to think of this. In particular, some "stages" contain (or can be thought of as-if they contain) another stage. Perhaps "operations" would be more useful here?

Reply:
Switched to "operations" for now. Not sure it's really right, but it's not horrible either.